[2021-05-25 21:31:03,090] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:28:29.460914+00:00 [queued]>
[2021-05-25 21:31:03,159] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:28:29.460914+00:00 [queued]>
[2021-05-25 21:31:03,159] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 21:31:03,159] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-25 21:31:03,159] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 21:31:03,206] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan_2020ToNow> on 2021-05-25T21:28:29.460914+00:00
[2021-05-25 21:31:03,229] {standard_task_runner.py:52} INFO - Started process 380 to run task
[2021-05-25 21:31:03,255] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan_2020ToNow', '2021-05-25T21:28:29.460914+00:00', '--job-id', '132', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmp7kikjln0', '--error-file', '/tmp/tmpupvfa60w']
[2021-05-25 21:31:03,264] {standard_task_runner.py:77} INFO - Job 132: Subtask Get_COVID19_Cases_For_Jordan_2020ToNow
[2021-05-25 21:31:03,424] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:28:29.460914+00:00 [running]> on host 4e0af5d83701
[2021-05-25 21:31:03,751] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan_2020ToNow
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T21:28:29.460914+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-25T21:28:29.460914+00:00
[2021-05-25 21:38:08,894] {local_task_job.py:77} ERROR - Received SIGTERM. Terminating subprocesses
[2021-05-25 21:38:09,132] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 380
[2021-05-25 21:38:09,137] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
